deploy/database/drop_databases.sql

```diff
@@ -1,2 +0,0 @@
-DROP DATABASE IF EXISTS buttonmen;
-DROP DATABASE IF EXISTS buttonmen_test;
```

Comment:
Why does this pull remove the drop_databases.sql script?  Is that intentional?  Do you need to merge from upstream?